resources/variants/structur3d_discov3ry1_complete_um2plus_0.20.inst.cfg

```diff
@@ -0,0 +1,14 @@
+[general]
+name = 0.20mm (Clear)
+version = 4
+definition = structur3d_discov3ry1_complete_um2plus
+
+[metadata]
+setting_version = 5
```

Comment:
Currently the Cura master version (set for 4.1 release) requires `setting_version = 7`. This field is used to detect whether the profile needs to be upgraded. And in your case, it needs to be upgraded. However it appears that your profiles don't use any of the settings whose name or functionality has changed in between, so we can just increment this number. I'll do that for you.